redwood-widget/src/androidMain/kotlin/app/cash/redwood/widget/RedwoodLayout.kt

```diff
@@ -27,7 +27,7 @@ import app.cash.redwood.ui.UiConfiguration
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 
-public open class RedwoodWidgetView(
+public open class RedwoodLayout(
```

Comment:
`RedwoodWidgetView` is very new, and hasn't been published as part of a release. We can therefore safely rename it without a deprecation path.